src/main/java/com/codegym/shopyy/model/Address.java

```diff
@@ -0,0 +1,39 @@
+package com.codegym.shopyy.model;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Table(name = "address")
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class Address {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private Long id;
+
+    @Column(name = "fullname")
```

Comment:
remove it because it will be auto-mapping with field of table with the same name